templates/careers/_filters.html

```diff
@@ -0,0 +1,26 @@
+<form action="" method="get" class="p-form p-form--inline">
+  <div class="p-form__group">
+    <label for="filter" aria-label="Filter by job type" class="p-form__label">Filter by</label>
+    <select class="js-filter p-form__control" id="filter" name="filter"
+      aria-label="Filter by job type">
+      <option value="all">All</option>
+      <option value="home-based">Home based</option>
+      <option value="office-based">Office based</option>
+      <option value="management">Management</option>
+      <option value="full-time">Full-time</option>
+      <option value="part-time">Part-time</option>
+    </select>
+  </div>
+  <div class="p-form__group">
+    <label for="sort" aria-label="Sort resources" class="p-form__label">Sort by</label>
+    <select class="js-sort p-form__control" id="sort" name="sort" aria-label="Sort resources">
+      <option>Date</option>
```

Comment:
I think these should have values.

Reply:
Done - tweaked the JS that was using this too.